qiskit/circuit/library/arithmetic/adder/reversibleadder.py

```diff
@@ -0,0 +1,82 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals
+
+"""Module-level docstring describing what the file content is."""
+
+
+from qiskit import QuantumRegister, ClassicalRegister, QuantumCircuit
+from qiskit import BasicAer
+from qiskit import execute
+
+
+
+class ReversibleAdder(QuantumCircuit):
+    """To implement QFT Adder.
+    
+    Based on the information given in [1].
+    
+    **References**
+    
+    [1] Thomas G.Draper, 2000. "Addition on a Quantum Computer"
+    `Journal https://arxiv.org/pdf/quant-ph/0008033.pdf`_
+    """
+    
+    def __init__(self, num_qubits_a: int, num_qubits_b: int, name='qft_adder') -> None:
+        """
+        Args:
+            num_qubits_a: The size of the first register.
+            num_qubits_b: The size of the second register.
+        """
+        # define the registers 
+        qr_a = QuantumRegister(num_qubits_a, name='a') 
+        qr_b = QuantumRegister(num_qubits_b, name='b') 
+        qr_cin = QuantumRegister(num_qubits_a,name='cin')
+        qr_cout = QuantumRegister(1, name='cout')
+        #qr_ans=ClassicalRegister(n+1,name='ans')
+	 
+        
+        # initialize the circuit
+        super().__init__(qr_a, qr_b,qr_cin,qr_cout, name=name)
+
+        qc_carry=QuantumCircuit(4,name='CARRY')
```

Comment:
We usually give the circuits lowercase names (or CamelCase), do you mind changing this?
```suggestion
        qc_carry=QuantumCircuit(4,name='Carry')
```

Reply:
No problem, will do